### PR TITLE
Add 'builder' call for Jacob

### PR DESCRIPTION
**Pull request by JacobJenness**
Adds a greeting for Jacob. The greeting has been implemented with similar functions, and maintains alphabetical order when printed.

### CODE REVIEW

**Review by schu4276 — DISMISSED** [review]
Looks good!

**Review by NicMcPhee — DISMISSED** [review]
This looks great!

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -20,6 +20,7 @@ public String generateOutput() {
     StringBuilder builder = new StringBuilder();
     builder.append(cassieSaysHello());
     builder.append(DanteSaysHello());
+    builder.append(jacobSaysHello());
```
Comment (day00090) [inline src/main/java/hellos/Hellos.java:23]: This is alphabetized correctly.

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -29,13 +30,17 @@ public String generateOutput() {
     return builder.toString();
   }
 
+  
+  private static String jacobSaysHello() {
+    return "Jacob says 'Salutations!'\n";
```
Comment (day00090) [inline src/main/java/hellos/Hellos.java:35]: This follows the format of the other greetings and appears to have passed the tests. It should be ready to go!

**Review by day00090 — APPROVED** [review]

**Review by NicMcPhee — APPROVED** [review]
Looks great!

**Pull request merged**